### DISCUSSION TITLE
Checking if CSS style is used somewhere in the HTML

### DISCUSSION THREAD

**Discussion by Tom712** [cypress-io/cypress#15193] [Questions and Help]
Is it possible to check if any element in the entire HTML page contains for example style : 'font-family'.
Or any other CSS style.
Like this: 
`<body>`

`<h1>This is a Heading</h1>`
`<p>This is a paragraph.</p>`

`<h1>This is another heading</h1>`
`<p style="font-family:courier;" >This is another paragraph.</p>`

`</body>`
`</html>`

And what if the CSS is in a separate file?

**Comment by 403-html** [comment]
hmm depends on the case, if you want to test for example that `p` tag if it has that style then you can achieve it by for example
```
cy.get('p').should('have.css', 'font-family', 'courier')
```
but if you have NO IDEA where (in which tag) this style parameter can be then you need to iterate through all of the tags, which (from my side) is not an efficient way to test things

**Reply by Tom712** [reply]
Yea... that's the problem. I'll have to iterate through them all.
